website/docs/guides/orchestration/webhooks/zapier-ms-teams.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Post to Microsoft Teams when a dbt Cloud job finishes"
```

Comment:
sugg shortening and removing dbt cloud since the main section references it

```suggestion
title: "Post to Microsoft Teams when a job finishes"
```